src/Illuminate/Database/Seeder.php

```diff
@@ -119,7 +136,7 @@ public function __invoke()
         }
 
         return isset($this->container)
-                    ? $this->container->call([$this, 'run'])
```

Comment:
I wouldn't change the formatting here.  let StyleCI pick up if this needs to be changed.